### ISSUE TITLE
Citations does NOT work in Sandbox

### ISSUE THREAD

**Issue by rgaiacs** [executablebooks/mystmd#487]
[Documentation says](https://mystmd.org/guide/citations) that "adding a markdown link to a [DOI](https://en.wikipedia.org/wiki/Digital_object_identifier)" will be parser as a citation.

Using

```
---
title: MWE
author: Raniere Silva
---

DOI links can be used to include citations <doi:10.5281/zenodo.6476040>.
```

in the [MyST Markdown Sandbox](https://mystmd.org/sandbox) produces

> MWE
>
> Raniere Silva
>
> DOI links can be used to include citations [doi:10​.5281​/zenodo​.6476040](https://doi.org/10.5281/zenodo.6476040).

![Screenshot 2023-07-14 at 16-05-45 MyST Markdown Sandbox](https://github.com/executablebooks/mystmd/assets/1506457/fcc60a39-6a81-4886-807d-22ff0d5e5060)

**Comment by rowanc1** [comment]
Thanks @rgaiacs! This hasn't been implemented yet as it is still tied together with some of the bibtex parsing code that only works on the file system!

Will get to this soon!

**Comment by lrq3000** [comment]
Yes please! This is the main reason I would like to implement mystmd parsing in another app (an Android editor), if it could parse citations, it would be I think the only open-source mobile editor to be able to do so!

**Comment by lrq3000** [comment]
Congratulations for the progress on the project overall! Any news on this feature?
